src/Psalm/Internal/LanguageServer/LanguageServer.php

```diff
@@ -200,16 +202,25 @@ public function initialize(
         return call(
             /** @return \Generator<int, true, mixed, InitializeResult> */
             function () use ($capabilities, $rootPath, $processId) {
+                $this->verboseLog("Initializing...");
+                $this->clientStatus('initializing');
```

Comment:
These client statuses should be known strings, so the client can use them and provide UI updates as needed, such as status appropriate icons, or extended status messages.